### DISCUSSION TITLE
EnhancedPBR and Skin MaterialType support dual lobe specular and Cavity Settings.

### DISCUSSION THREAD

**Discussion by wangwei46810** [o3de/o3de#14611] [General]
We have added Dual Lobe Specular and Cavity settings on EnhancedPBR and Skin Materials in O3DE in order to enhances area highlights on face and details of skin.  And now , we would like to contribute it to the community. Wish any feedback or suggestions, thanks. 

**Implementation of the Dual Lobe Specular Layer:** 

- Different from Unreal Engine, we added a "Dual Normal Texture" and "Dual Roughness Texture" in order for artists to load another normal and roughness textures to acquire more details for further blend. Besides, artists could modify Normal facter,   Lower Bound and Upper Bound of  Dual Roughness and their Blend Factor. Traditional highlight effect was mixed with dual specualr to enhance area highlights. 

**Implementation of the Cavity Layer :**

- Artists could add a cavity map into the material and adjust the factors . We sampled the Cavity map , multified it with Cavity factors. Traditional specularF0 channel was multified with Cavity parameters to enhanced highlight details of material.

**Comment by TheRexRed** [comment]
The Daz Horse 3 has dual lobe layer textures on the mane and other hair items on the horse. Diffeomorphic to Blender will not allow me to put a texture that will work on these dual lobe hair items. How do I work around this problem? Thanks in advance for any suggestions.

**Comment by AMZN-alexpete** [comment]
Related https://github.com/o3de/o3de/pull/16677 "[Dual Specular for Skin](https://github.com/o3de/o3de/pull/16677/files#top)"
